src/dom/Draggable.js

```diff
@@ -1,3 +1,12 @@
+import {Evented} from '../core/Events';
+import {touch} from '../core/Browser';
+import {on, off, preventDefault} from '../dom/DomEvent';

```

Comment:
We might want to do `on as onDOMEvent` or similar (I've used this earlier in rollup2 branch), because `on` without a namespace is too generic.